src/helpers/radspec.js

```diff
@@ -10,21 +10,21 @@ const makeUnknownFunctionNode = (methodId) => ({
 })
 
 const getSig = (fn) =>
-  keccak256(fn).substr(0, 10)
+  ethers.utils.keccak256(ethers.utils.toUtf8Bytes(fn)).substr(0, 10)
```

Comment:
Hmm this feels a bit weird, does ethers have a `encodeFunctionSignature()` util that takes the function signature as input?